backend/cmd/headlamp.go

```diff
@@ -937,6 +952,12 @@ func (c *HeadlampConfig) getClusters() []Cluster {
 
 	for _, context := range contexts {
 		context := context
+
+		// Stateless clusters should not be visible to other users.
```

Comment:
Lets call it Dynamic for consistency.
```suggestion
		// Dynamic clusters should not be visible to other users.
```